app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap

```diff
@@ -266,7 +301,25 @@ exports[`Metadata should render links with icons if propery has url and icon 1`]
             "translateContext": "oneTranslateContext",
             "type": "relationship",
             "url": "url",
-            "value": "string value",
+            "value": Array [
+              Object {
+                "0": "s",
+                "1": "t",
+                "10": "u",
+                "11": "e",
+                "2": "r",
+                "3": "i",
+                "4": "n",
+                "5": "g",
+                "6": " ",
+                "7": "v",
+                "8": "a",
+                "9": "l",
+                "value": <Connect(RelationshipLink)
+                  propValue="string value"
+                />,
```

Comment:
Same as above... why is the string being made into an array?

Reply:
Fixed